packages/gatsby-theme-newrelic/gatsby-node.js

```diff
@@ -216,6 +216,24 @@ exports.createResolvers = ({ createResolvers }, themeOptions) => {
         },
       },
     },
+    Quickstarts: {
+      relatedResources: {
```

Comment:
This block seems pretty similar to lines 202-217...maybe we can move this to a constant? That way we don't need to update things like `defaultValue` in multiple places.

Reply:
ah yes, that's a good idea!